janusgraph-test/pom.xml

```diff
@@ -30,6 +30,13 @@
     </repositories>
 
     <dependencies>
+        <!-- Enforce a classpath ordering constraint to ensure log4j is used for logging. -->
```

Comment:
I hit this error from Travis
```
The job exceeded the maximum log length, and has been terminated.
```
and found out that the log4j settings were not being used because of a conflict on the available SLF4J available at run time. This change enforces the use of log4j which make the settings in `log4j.properties` being picked up.

The logs are much quieter now, the full DEBUG output is being appended to a file. Let me know if this is the right approach.

Reply:
Yeah this is a known issue.